cdap-app-templates/cdap-etl/cdap-etl-lib/widgets/AmazonSQS.json

```diff
@@ -0,0 +1,41 @@
+{
+  "metadata": {
+    "spec-version": "1.0",
+    "artifact-version": "3.3.0-SNAPSHOT",
```

Comment:
Also, @ajainarayanan please modify the specification document if you make this change. Also move the specification document to wiki.cask.co.